src/utils/FontFactory.ts

```diff
@@ -0,0 +1,37 @@
+import { zipFolder, addConfigFile } from './tools'
+
+import { tempDirectory } from '../common/constants'
+
+class FontFactory {
+    private icons: Array<string>
+    private timestamp: number
+    private theme: string
+
+    constructor (icons: Array<string>, timestamp: number, theme: string) {
+      this.icons = icons
+      this.timestamp = timestamp
+      this.theme = theme
+    }
+
+    gruntCdIcons () {
+      try {
+        let command = ''
+        for (let i = 0; i < this.icons.length; i++) {
+          command += `--extended_src=${this.theme}/${this.icons[i]}.svg `
+        }
+        return `grunt -b ./src --dist=${this.timestamp} ${command} --outlined=${this.theme === 'svg-outlined'}`
+      } catch (err) {
+        throw new Error(`Some error occurred while generating Icons set command: ${err}`)
+      }
+    }
```

Comment:
the `gruntCdIcons()` generates the command and `generateFIles` actually doesnt generate files, it is just zipping the already generated files. as far as i remember we had a package node-cli to run the grunt command, can we move that functions here to to this class itself?

Reply:
Yes, It make more sense, to be moved to the class itself. I will work on it